_pages/en_US/seedminer.txt

```diff
@@ -75,19 +80,30 @@ There are several different methods that build on the Seedminer exploit, so even
 
 ___
 
-#### Seedminer + unSAFE_MODE
+#### Seedminer + BannerBomb3 + unSAFE_MODE
 
 This method of using Seedminer for further exploitation uses your `movable.sed` file to take advantage of exploits in the SAFE_MODE firmware present on all 3DS units.
 
 This method is compatible with all regions, though the CHN region is not covered by this guide.
 
-Continue to [Installing boot9strap (USM)](installing-boot9strap-(usm))
+Continue to [Installing boot9strap (unSAFE_MODE](installing-boot9strap-(usm))
+{: .notice--primary}
+
+___
+
+#### Seedminer + PicHaxx + unSAFE_MODE
+
+This method of using Seedminer for further exploitation uses your `movable.sed` file to gain access to the Homebrew Launcher using the PicHaxx exploit for the purpose of injecting an exploited WiFi profile. This method requires you to already own (or download) the free "Pokemon Picross" game from the eShop.
+
+This method is only recommended if your console failed the DSiWare Management test
+
+Continue to [Installing boot9strap (slotTool)](installing-boot9strap-(slottool))
 {: .notice--primary}
 
 ___
 
 #### Legacy Methods
 
-These methods are outdated and not recommended unless otherwise indicated.
+These methods are not recommended and are only intended for those who cannot access safemode
```

Comment:
Wording is inconsistent (Safe Mode is spaced + capitalized), and should be a complete sentence.